### DISCUSSION TITLE
Authenticating Server-Rendered next js 13.4

### DISCUSSION THREAD

**Discussion by easy-why-not** [vercel/next.js#50051] [App Router]
Hi!
 In previous versions of Next js it was possible to produce Authenticating Server-Rendered using

export const checkAuth = async (ctx: GetServerSidePropsContext) => {
  const { access } = nookies.get(ctx);
  axios.defaults.headers.Authorization = `${access}`;
  try {
    const user = await UserApi.getMe();
    return {
      props: { user }
    };
  } catch (err) {
    return {
      redirect: {
        destination: 'login',
        permanent: false
      }
    };
  }
};

How to do it on Next js 13.4?
Thanks

**Comment by Davicastro19** [comment]
I would also like to know
